tests/test_excelparser/result_ontology/fromexcelonto_old.ttl

```diff
@@ -0,0 +1,143 @@
+@prefix : <http://emmo.info/emmo/domain/onto#> .
```

Comment:
I guess that this file shouldn't be checked in.

Reply:
Consider to squash merge this PR to not increase the repo size unnecessary. This will only create one commit in the git history, reducing possibilities for later debugging with get bisect

Reply:
I have renemed this so that it shows that it is adifferent excelfile and that is tested and should not be deleted.